### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by izzyhope58**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [ X] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [X ] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [X ] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Comment by izzyhope58** [comment]
Is there an issue with how I submitted this? I'm not sure what checks have failed or why it didn't work

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -23009,3 +23009,25 @@
         <Description>Autosplitter, load remover and game time</Description>
     </AutoSplitter>
 </AutoSplitters>
+    <AutoSplitter>
```
Comment (wooferzfg) [inline LiveSplit.AutoSplitters.xml:23012]: These need to be inside the `AutoSplitters` tag

**Comment by izzyhope58** [comment]
Don't quite understand what you mean, but I think I fixed the formating (sorry, I'm really bad at github and formatting stuff)

**Comment by wooferzfg** [comment]
You still need the `</AutoSplitters>` tag at the end of the file, like before

File: LiveSplit.AutoSplitters.xml [hunk 2]
```diff
@@ -23008,4 +23008,26 @@
...
+        <Games>
+            <Game>Diseny Infinity: Inside Out</Game>
+            <Game>Inside Out</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/izzyhope58/ASL-Scripts/refs/heads/main/InsideOut.asl</URL>
+        </URLs>
+        <Type>Script</Type>
+        <Description>Autosplitter & Loadremover. Created by IzzyHope58 with help from AndetSR</Description>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:23020]:
```suggestion
        <Description>Autosplitter and Loadremover. Created by IzzyHope58 with help from AndetSR</Description>
```

File: LiveSplit.AutoSplitters.xml [hunk 3]
```diff
@@ -23008,4 +23008,26 @@
...
+    </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>The Incredibles</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/izzyhope58/ASL-Scripts/refs/heads/main/TheIncredibles.asl</URL>
+        </URLs>
+        <!-- Type of auto splitter (Script/Component). -->
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:23029]:
```suggestion
```

File: LiveSplit.AutoSplitters.xml [hunk 4]
```diff
@@ -23008,4 +23008,26 @@
...
+        <Type>Script</Type>
+        <Description>Autosplitter & Loadremover for the PC version. Created by IzzyHope58</Description>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:23031]:
```suggestion
        <Description>Autosplitter and Loadremover for the PC version. Created by IzzyHope58</Description>
```

**Pull request merged**